moderation/logging/translations/en.yml

```
@@ -24,7 +24,8 @@ message_id: Message ID
 created_at: Created At
 
 member_joined_server: "{} ({}) just joined the server!"
-member_left_server: "**{}** just left the server!"
+member_left_server_with_joined: "**{}** ({}) just left the server! Latest join was at {}"
```

```suggestion
member_left_server_with_joined: "**{}** ({}) just left the server! The most recent join occurred at {}."
```
Or even better: "The most recent join occurred on 02/24/2023 at 15:00:00."
This template makes use of the correct prepositions.